app/views/layouts/provider/login.html.slim

```diff
@@ -8,6 +8,7 @@ html lang="en"
     = render 'provider/analytics'
     = stylesheet_link_tag 'loginPageFonts'
     = javascript_pack_tag 'pf4BaseStyles'
+    = javascript_pack_tag 'LoginPage'
```

Comment:
I think this has to be loaded after the body is created, not in the head.

Reply:
But it will wait for `DOMContentLoaded`

Reply:
Yes exactly otherwise it will raise the same issue:

![image](https://user-images.githubusercontent.com/64276/59856512-53edde80-9377-11e9-89d4-8c12b1ed7ca4.png)

Reply:
And it is not the only page that should be touched, it is wherever shim.js is used in the react components.